package.json

```diff
@@ -1,24 +1,26 @@
 {
-  "name": "bitdao-token-contract",
+  "name": "bitdao-governance",
   "version": "1.0.0",
-  "description": "The BitDAO contract as deployed on Mainnet",
+  "description": "BitDAO Governance framework contracts",
   "private": true,
   "directories": {
     "test": "test"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/bitdao-io/bitdao-token-contract.git"
+    "url": "git+https://github.com/bitdao-io/governance.git"
   },
   "keywords": [],
   "author": "",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "hardhat compile",
-    "clean": "hardhat clean",
-    "format": "prettier ./**/*.ts --write",
+    "compile": "npx hardhat compile",
```

Comment:
I'd recommend against changing the build command to compile, reasoning:
- consistency with TypeScript/JavaScript projects; a build target is the common one used for building the project.
- compile is an inaccurate term; While Solidity is compiled, TypeScript is transpiled, while similar is not equivalent, specificity can be valuable.
- build target is supported as default script by Npm, while compile is not https://docs.npmjs.com/cli/v6/commands/npm-build

A solution here can be to have both `build` and `compile` targets 🙂